### PR TITLE
Reorganise to quarto

### PR DESCRIPTION
**Pull request by seabbs**
This PR is optional as its reorganises to quarto. For discussion.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -6,23 +6,20 @@ output: github_document
...
+To add a lesson, add a folder in the `sessions` folder that contains a index.qmd`, and the edit `sessions.qmd` to add it to the schedule.
 
 ## Local testing
 
 The `html` pages can be generated locally using the function
 
-```{r render, eval = FALSE}
-rmarkdown::render_site("Rmd/")
+``` bash
+quarto render
```
Comment (sbfnk) [inline README.Rmd:20]:
```suggestion
``` {r render, eval = FALSE}
quarto::quarto_render()
```

maybe simpler as R only workflow?

File: _quarto.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+project:
+  type: website
+
+website:
+  title: "NFIDD"
+  site-url: https://nfidd.github.io/nfidd/
+  description: "Nowcasting and forecasting infectious disease dynamics"
+  image: images/logo.png
+  favicon: images/favicon-32x32.png
```
Comment (sbfnk) [inline _quarto.yml:9]: take out whilst we don't have a logo?

```suggestion
```

File: _quarto.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      category: 'Website discussions'
+      mapping: title
+      reactions-enabled: true
+  navbar:
+    left:
+      - text: Sessions
+        href: sessions.qmd
+      - text: "Learning objectives"
+        href: learning_objectives.qmd
+      - text: "Getting setup for the course"
```
Comment (sbfnk) [inline _quarto.yml:23]:
```suggestion
      - text: "Getting set up for the course"
```

File: _quarto.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        href: getting-setup.qmd
```
Comment (sbfnk) [inline _quarto.yml:24]:
```suggestion
        href: getting-set-up.qmd
```

File: _quarto.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  description: "Nowcasting and forecasting infectious disease dynamics"
+  image: images/logo.png
+  favicon: images/favicon-32x32.png
+  open-graph: true
+  comments:
+    giscus:
+      repo: nfidd/nfidd
+      category: 'Website discussions'
+      mapping: title
+      reactions-enabled: true
```
Comment (sbfnk) [inline _quarto.yml:16]: later we deactivate them everywhere? Perhaps leave discussions to the github site (open for discussion)

```suggestion
  comments: false
```

File: index.qmd [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Welcome to the Nowcasting and forecasting infectious disease dynamics course site"
```
Comment (sbfnk) [inline index.qmd:2]:
```suggestion
title: "Nowcasting and forecasting infectious disease dynamics"
```
rest is superfluous?

File: index.qmd [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+site: distill::distill_website
+comments: false
+about:
+  template: jolla
+  image-width: 6em
+  image-shape: rectangle
+  image: images/logo.png
```
Comment (sbfnk) [inline index.qmd:9]:
```suggestion
```

remove whilst we don't have a logo?

File: sessions/1-R-Stan-and-statistical-concepts/index.qmd [hunk 8]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Session 1: R, Stan, and statistical concept background"
```
Comment (sbfnk) [inline sessions/1-R-Stan-and-statistical-concepts/index.qmd:2]:
```suggestion
title: "Session 1: R, Stan, and statistical concepts"
```

File: sessions/4-using-delay-distributions-to-model-the-data-generating-process-of-an-epidemic/index.qmd [hunk 9]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Session 4: Using delay distributions to model the data generating process of an epidemic"
```
Comment (sbfnk) [inline sessions/4-using-delay-distributions-to-model-the-data-generating-process-of-an-epidemic/index.qmd:2]:
```suggestion
title: "Session 4: Using delay distributions to model the data generating process"
```

**Review by sbfnk — CHANGES_REQUESTED** [review]
Looks great, thank you!

I'm not particularly clear on benefits of quarto vs. rmarkdown except quarto is newer (which in itself may be a benefit), but happy to go ahead with it if you think sensible.

Suggested a few re-wordings and to take out the epinowcast logo (/possibly replace with our own one later?).

File: _quarto.yml (hunk 2 above)
Comment (seabbs) [reply _quarto.yml:9]: Sorry I missed this - I meant it just as a placeholder.

File: _quarto.yml (hunk 5 above)
Comment (seabbs) [reply _quarto.yml:14]: We don't have them turned off everywhere we just have them turned off on not sessions. This is nice as the comments are posts on the github discussion. That being said I see why we might not want comments so it's definitely a judgement call.

**Comment by seabbs** [comment]
> I'm not particularly clear on benefits of quarto vs. rmarkdown except quarto is newer (which in itself may be a benefit), but happy to go ahead with it if you think sensible.

It being newer and having slightly cleaner styling + more functionality is kind of the only benefit. I'm pretty easy either way and I think a lot depends on what the workflow is now like in Rstudio using `qmd` docs (i.e. is it like a traditional `Rmd` notebook). My expectation is yes to that (@kathsherratt do you know?)

**Review by sbfnk — APPROVED** [review]

**Comment by kathsherratt** [comment]
Fancy! I do not know quarto workflows and haven't used it, sorry.

**Pull request merged**